src/PermissionManagerInterface.php

```diff
@@ -31,6 +32,25 @@ public function getPermissionList($entity_type_id, $bundle_id);
    * @return array
    *   An array of permission names and descriptions.
    */
-  public function generateCrudPermissionList($group_content_entity_type_id, $group_content_bundle_id);
+  public function generateEntityOperationPermissionList($group_content_entity_type_id, $group_content_bundle_id);
+
+  /**
+   * Returns permissions that are enabled by default for the given role.
+   *
+   * @param string $group_entity_type_id
+   *   The entity type ID of the group for which to return permissions.
+   * @param string $group_bundle_id
+   *   The bundle ID of the group for which to return permissions.
+   * @param array $group_content_bundle_ids
```

Comment:
I'm not clear about `$group_content_bundle_ids` - why do we need this in this context?